vendor/github.com/Azure/azure-sdk-for-go/services/mediaservices/mgmt/2018-07-01/media/accountfilters.go

```diff
@@ -0,0 +1,487 @@
+package media
```

Comment:
can we vendor these files in? update: this requires rebasing on top of master - so I'm going to push a commit for this